packages/samples/mvp-dapp/src/app.controller.ts

```diff
@@ -3,18 +3,22 @@ import { HexString, Script, helpers } from '@ckb-lumos/lumos'
 import { ActorReference, Manager, ProviderKey, UpdateStorageValue } from '@ckb-js/kuai-models'
 import { NotFound, BadRequest } from 'http-errors'
 import { appRegistry } from './actors'
-import { Load } from './views/load.view'
-import { Read } from './views/read.view'
 import { OmnilockModel } from './omnilock/omnilock.model'
 import { computeScriptHash } from '@ckb-lumos/base/lib/utils'
 import { RecordModel, StoreType } from './record/record.model'
 import { DAPP_DATA_PREFIX } from './const'
+import { Tx } from './views/tx.view'
 
 const router = new KuaiRouter()
 function createCellPattern(lock: Script) {
   return (value: UpdateStorageValue) => {
     const cellLock = value.cell.cellOutput.lock
-    return cellLock.args === lock?.args && cellLock.codeHash === lock?.codeHash && cellLock.hashType === lock?.hashType
+    return (
+      cellLock.args === lock?.args &&
+      cellLock.codeHash === lock?.codeHash &&
+      cellLock.hashType === lock?.hashType &&
+      value.cell.data === '0x'
```

Comment:
If the `data` is obligatory to be `0x`, the method name may be confusing because it's specific for plain omnilock cells

Reply:
Of cause, it's the store to save plain omnilock cells. Add filter `data==='0x'` to avoid used cells that have stored some data when calling `claim`.